modules/openapi-generator/src/main/resources/dart/libraries/dio/pubspec.mustache

```diff
@@ -4,7 +4,12 @@ description: {{pubDescription}}
 homepage: {{pubHomepage}}
 
 environment:
+{{#useBuiltValue}}
   sdk: '>=2.12.0 <3.0.0'
+{{/useBuiltValue}}
+{{#useJsonSerializable}}
+  sdk: '>=2.14.0 <3.0.0'
```

Comment:
Whats the reasoning here?
@agilob Can we do this?

Reply:
`openapi depends on json_serializable >=6.0.0 which requires SDK version >=2.14.0 <3.0.0`

Reply:
Fine by me

Reply:
Yep sorry I missed that in my comment above. `json_annotation 4.2`, which we are using for the better enum support, requires Dart 2.14.